x/auth/ante/expected_keepers.go

```diff
@@ -18,3 +18,12 @@ type AccountKeeper interface {
 type FeegrantKeeper interface {
 	UseGrantedFees(ctx sdk.Context, granter, grantee sdk.AccAddress, fee sdk.Coins, msgs []sdk.Msg) error
 }
+
+type CircuitBreakerKeeper interface {
+	// Check if the circuit breaker is open
+	IsCircuitOpen(ctx sdk.Context) bool
+	// Open the circuit breaker
+	OpenCircuit(ctx sdk.Context)
+	// Close the circuit breaker
+	CloseCircuit(ctx sdk.Context)
```

Comment:
I'm pretty sure we want to filter on message name and I don't think we need to open or close the circuit breaker in the ante handler. Maybe this is sufficient:
```go
    IsAllowed(ctx sdk.Context, msgName string) bool
```